CreeDictionary/search_quality/__init__.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
```

Comment:
Since mypy was updated to 0.800, this is no longer needed!

```suggestion
```

Reply:
`CreeDictionary/search_quality/analyze_results.py:31: error: X | Y syntax for unions requires Python 3.10`

Reply:
Do you need it in both `__init__.py` _and_ `analyze_results.py`?

Reply:
Nope, just in `analyze_results.py`.